src/twisted/python/rebuild.py

```diff
@@ -210,7 +210,10 @@ def rebuild(module, doLog=1):
         log.msg("")
         log.msg(f"  (fixing   {str(module.__name__)}): ")
     modcount = 0
-    for mk, mod in sys.modules.items():
+    # note: sys.modules can change throughout iteration
```

Comment:
Very minor comment.

I am not sure that we need "note:" ... basically, all comments are added as "a note" :)

```suggestion
    # sys.modules can change throughout iteration
```